apps/dashboard/app/(app)/logs/components/charts/index.tsx

```diff
@@ -0,0 +1,131 @@
+"use client";
+import {
+  type ChartConfig,
+  ChartContainer,
+  ChartTooltip,
+  ChartTooltipContent,
+} from "@/components/ui/chart";
+import type { LogsTimeseriesDataPoint } from "@unkey/clickhouse/src/logs";
+import { addMinutes, format } from "date-fns";
+import { useEffect, useState } from "react";
+import { Bar, BarChart, CartesianGrid, ResponsiveContainer, XAxis, YAxis } from "recharts";
+import { useFetchTimeseries } from "./hooks";
+
+const chartConfig = {
+  success: {
+    label: "Success",
+    color: "hsl(var(--chart-3))",
```

Comment:
fine for now, but we gotta switch these to the new 12 step color scale
https://engineering.unkey.com/design/colors

Reply:
Will do when we move to design of this page